mount/src/assets/comment.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import SortIcon from '@mui/icons-material/Sort';
+
+
+export default class Posts extends React.Component {
```

Comment:
import Component from react library at line no 1 , here directly use Component not React.component